internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go

```diff
@@ -68,7 +71,9 @@ func (l *Manager) Wakeup(ctx context.Context, appID api.AppID) error {
 	item.nextRequestAfter = now.Add(Interval)
 
 	// Send the notification
-	if _, err := l.api.WakeupApp(appID).Send(ctx); err != nil {
+	_, err := l.api.WakeupApp(appID).Send(ctx)
+	// If it does not succeed but app is currently stopping do not log it as error, log only other errors
```

Comment:
I can tell as much from the code. What I want in the comment is the reason why we're doing this.

Reply:
3979510edd27e6305524d2e04f4b316a3a8e732f